pkg/agent/route/route_windows.go

```diff
@@ -383,7 +398,7 @@ func (c *Client) syncRoute() error {
 	gwAutoconfRoute := &util.Route{
 		LinkIndex:         c.nodeConfig.GatewayConfig.LinkIndex,
 		DestinationSubnet: c.nodeConfig.PodIPv4CIDR,
-		GatewayAddress:    c.nodeConfig.GatewayConfig.IPv4,
+		GatewayAddress:    net.IPv4zero,
```

Comment:
So route to a local subnet should use 0.0.0.0 as gateway?

Reply:
The gateway of the route added by Windows is 0.0.0.0.